docs/podman-inspect.1.md

```diff
@@ -96,6 +96,11 @@ overlay
 size:   4405240
 ```
 
+```
+podman inspect --latest --format {{.EffectiveCaps}}
+[CAP_CHOWN CAP_DAC_OVERRIDE CAP_FSETID CAP_FOWNER CAP_MKNOD CAP_NET_RAW CAP_SETGID CAP_SETUID CAP_SETFCAP CAP_SETPCAP CAP_NET_BIND_SERVICE CAP_SYS_CHROOT CAP_KILL CAP_AUDIT_WRITE]
+```
+
```

Comment:
The example at line 38 doesn't show these fields.  Can we update that?   Being careful to keep the other examples valid in comparison.